### ISSUE TITLE
Gatekeeper::getGroupByName() - throws wrong exception

### ISSUE THREAD

**Issue by Ser5** [psecio/gatekeeper#21]
I accidentally wrote `Gatekeeper::getGroupByName()` instead of `Gatekeeper::findGroupByName()` and it gave me wonderful we-need-to-go-deeper error:

```
PHP Fatal error:  Class 'Psecio\Gatekeeper\Handler\Exception\ModelNotFoundException' not found in /blah/htdocs/core/lib/vendor/psecio/gatekeeper/src/Psecio/Gatekeeper/Handler/FindBy.php on line 63
```

I presume, \Psecio\Gatekeeper\Exception\ModelNotFoundException class is meant.

**Comment by enygma** [comment]
Ah, you are correct - that `Handler` must have slipped in there on accident.

**Comment by Ser5** [comment]
Forgot to note that the wrong exception appears in three places in that file, not just line 63.

**Comment by enygma** [comment]
This has been corrected in 9f26415. Closing issue.
